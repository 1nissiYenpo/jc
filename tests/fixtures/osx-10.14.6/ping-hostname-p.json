{"destination_ip": "151.101.1.67", "data_bytes": 56, "pattern": "0xff", "destination": "cnn.com", "packets_transmitted": 3, "packets_received": 3, "packet_loss_percent": 0.0, "round_trip_ms_min": 89.422, "round_trip_ms_avg": 118.033, "round_trip_ms_max": 147.964, "round_trip_ms_stddev": 23.918, "responses": [{"type": "reply", "bytes": 64, "response_ip": "151.101.1.67", "icmp_seq": 0, "ttl": 59, "time_ms": 89.422}, {"type": "reply", "bytes": 64, "response_ip": "151.101.1.67", "icmp_seq": 1, "ttl": 59, "time_ms": 116.712}, {"type": "reply", "bytes": 64, "response_ip": "151.101.1.67", "icmp_seq": 2, "ttl": 59, "time_ms": 147.964}]}
