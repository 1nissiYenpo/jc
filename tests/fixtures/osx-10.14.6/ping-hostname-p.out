PATTERN: 0xff
PING cnn.com (151.101.1.67): 56 data bytes
64 bytes from 151.101.1.67: icmp_seq=0 ttl=59 time=89.422 ms
64 bytes from 151.101.1.67: icmp_seq=1 ttl=59 time=116.712 ms
64 bytes from 151.101.1.67: icmp_seq=2 ttl=59 time=147.964 ms

--- cnn.com ping statistics ---
3 packets transmitted, 3 packets received, 0.0% packet loss
round-trip min/avg/max/stddev = 89.422/118.033/147.964/23.918 ms
